cmd/gpud/command/run.go

```diff
@@ -30,15 +30,20 @@ func cmdRun(cliContext *cli.Context) error {
 
 	var zapLvl zap.AtomicLevel = zap.NewAtomicLevel() // info level by default
 	if logLevel != "" && logLevel != "info" {
-		lCfg := log.DefaultLoggerConfig()
 		var err error
 		zapLvl, err = zap.ParseAtomicLevel(logLevel)
 		if err != nil {
 			return err
 		}
+	}
+	if logFile != "" {
+		log.Logger = log.CreateLoggerWithLumberjack(logFile, 128, zapLvl.Level())
+	} else {
+		lCfg := log.DefaultLoggerConfig()
 		lCfg.Level = zapLvl
 		log.Logger = log.CreateLogger(lCfg)
 	}
+
```

Comment:
We should merge them together and pass the logfile to the log package. If the logfile is empty, we should print the logs to stdout/stderr. Otherwise, we should output logs to both the file and stdout/stderr. Additionally, we should add a flag to disable output to stdout/stderr (by default, it should be enabled). In gpud.service, we should only print to the file.